easybuild/easyconfigs/p/PartitionFinder/PartitionFinder-2.1.1-intel-2019b-Python-2.7.16.eb

```
@@ -0,0 +1,49 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Tarball'
+
+name = 'PartitionFinder'
+version = '2.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.robertlanfear.com/partitionfinder'
+description = """PartitionFinder 2 is a Python program for simultaneously
+choosing partitioning schemes and models of molecular evolution for phylogenetic
+analyses of DNA, protein, and morphological data. You can PartitionFinder 2
+before running a phylogenetic analysis, in order to decide how to divide up
+your sequence data into separate blocks before analysis, and to simultaneously
+perform model selection on each of those blocks."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+github_account = 'brettc'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['ccf3718996ee6ca496909b4b97d2b075028e0543eba3bc47a8c14b689c84e061']
+
+dependencies = [
+    ('Python', '2.7.16'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('scikit-learn', '0.20.4', versionsuffix),
+    ('PyTables', '3.5.2', versionsuffix),
+    ('pyparsing', '2.4.6', versionsuffix),
+]
+
+postinstallcmds = ["cd %(installdir)s && chmod a+x PartitionFinder*"]
+fix_python_shebang_for = ["*.py"]
+
+sanity_check_paths = {
+    'files': ['PartitionFinderMorphology.py', 'PartitionFinderProtein.py', 'PartitionFinder.py'],
+    'dirs': ['partfinder'],
+}
+
+sanity_check_commands = [
+    'PartitionFinder.py -p %(parallel)s %(installdir)s/examples/nucleotide/test.phy',
+    'PartitionFinderProtein.py -p %(parallel)s %(installdir)s/examples/aminoacid/test.phy',
+    'PartitionFinderMorphology.py -p %(parallel)s %(installdir)s/examples/morphology/test.phy --raxml',
```

@Darkless012 The style check we have is sometimes a bit strict w.r.t. use of template values, it wants you to use %(name)s.py` rather than `PartitionFinder.py`.

This wasn't an issue before because you were using `partitionfinder` as name.

The easiest way forward is to just make it happy, as follos:

```suggestion
    '%(name)s.py -p %(parallel)s %(installdir)s/examples/nucleotide/test.phy',
    '%(name)sProtein.py -p %(parallel)s %(installdir)s/examples/aminoacid/test.phy',
    '%(name)sMorphology.py -p %(parallel)s %(installdir)s/examples/morphology/test.phy --raxml',

---

Thanks, I was suspecting something like this. +1 for this feature :)